core/src/main/java/com/linecorp/armeria/client/retry/RetryStrategy.java

```diff
@@ -34,19 +36,21 @@
 @FunctionalInterface
 public interface RetryStrategy<I extends Request, O extends Response> {
 
+    Backoff defaultBackoff = Backoff.of(Flags.defaultBackoffSpec());
+
     /**
      * A {@link RetryStrategy} that defines a retry should not be performed.
      */
     static <I extends Request, O extends Response> RetryStrategy<I, O> never() {
         return new RetryStrategy<I, O>() {
             @Override
-            public CompletableFuture<Boolean> shouldRetry(I request, O response) {
-                return CompletableFuture.completedFuture(false);
+            public CompletableFuture<Optional<Backoff>> shouldRetry(I request, O response) {
```

Comment:
@imasahiro Sorry, I have to change the api again in order to meet the needs in #655